docs/how-to/configure-update-time-range.md

```diff
@@ -0,0 +1,45 @@
+# How to configure update-time-range
+
+### Configure update-time-range
+
+To configure when Jenkins-k8s looks up for patches and automatically update, you can apply
```

Comment:
How about `Use the `update-time-range` configuration to set the time interval when `jenkins-k8s` automatically applies the latest patches for the current LTS version.`?

Reply:
Updated! Thank you!